airflow/jobs/triggerer_job.py

```diff
@@ -483,9 +483,14 @@ async def create_triggers(self):
         while self.to_create:
             trigger_id, trigger_instance = self.to_create.popleft()
             if trigger_id not in self.triggers:
+                task_instance: TaskInstance = trigger_instance.task_instance
+                dag_id = task_instance.dag_id
+                execution_date = task_instance.execution_date
```

Comment:
Since `execution_date` is disfavoured, it is better to log the run ID instead.

Reply:
I was confused which one to choose, I'll change it